src/ZprimeSelectionModule.cxx

```diff
@@ -141,7 +144,13 @@ bool ZprimeSelectionModule::process(Event & event){
   jetlepton_cleaner->process(event);
   jetER_smearer->process(event);
   jet_cleaner1->process(event); // jets w/ pt>25 GeV for lepton-2Dcut
-  bool pass_twodcut = twodcut_sel->passes(event); // lepton 2D-cut
+  //Trigger selection
+  bool ele_Trigger_selection = ele_Trigger->passes(event);
+  bool mu_Trigger_selection = mu_Trigger->passes(event);
+  if (!ele_Trigger_selection || !mu_Trigger_selection || (mu_Trigger_selection && ele_Trigger_selection)) return false;
```

Comment:
I don't think this is a good idea: you can easily mess up by changing the trigger config but not the selection config. Also, you don't really save by that appraoch: the only thing you save is one triggerselection initialization at startup which is negligible in terms of both CPU time and runtime.